flash/image/embedding/losses/vissl_losses.py

```diff
@@ -27,7 +26,6 @@
 
 
 def get_loss_fn(loss_name: str, cfg: AttrDict):
-    set_cpu_device()
     loss_fn = LOSS_REGISTRY[loss_name](cfg)
```

Comment:
The `__init__` function of SimCLR loss criterion pre-computes some masks, and setting CPU device explicitly here led us to get devices mismatch error for further calculates (masks were on CPU, while other computations were on CUDA).